iree/compiler/Dialect/Flow/Transforms/HLOToHLOPreprocessing.cpp

```diff
@@ -34,6 +34,11 @@ static llvm::cl::opt<bool> extractPadFromConv(
     llvm::cl::desc("Extract padding attributes from conv op"),
     llvm::cl::init(true));
 
+static llvm::cl::opt<bool> conv1x1toDot(
+    "iree-1x1-conv-to-dot",
+    llvm::cl::desc("Rewrites mhlo.conv with 1x1 filter into mhlo.dot"),
+    llvm::cl::init(false));
```

Comment:
any reason not to have this always on? it seems awesome :)

Reply:
The introduced reshape op composes in a way that cases a bug in linalg lowering see:#2883